### PR TITLE
[1.8.x] Renamed RemovedInDjango20Warning to RemovedInDjango110Warning.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: django/utils/deprecation.py [hunk 1]
```diff
@@ -2,9 +2,12 @@
 import warnings
 
 
-class RemovedInDjango20Warning(PendingDeprecationWarning):
+class RemovedInDjango110Warning(PendingDeprecationWarning):
     pass
 
+# for backwards compatibility in Django 1.8.3
+RemovedInDjango20Warning = RemovedInDjango110Warning
```
Comment (loic) [inline django/utils/deprecation.py]: I think I would do `RemovedInDjango20Warning = PendingDeprecationWarning` so we don't give it a meaning it doesn't really have.

**Comment by loic** [comment]
Regarding master, what do you have in mind? Cherry pick this commit onto master and solve conflict?

**Comment by loic** [comment]
https://gist.github.com/14aa71293c2cf2d918e8

**Comment by timgraham** [comment]
Yes for master (+add renames for 21Warning)

**Pull request merged**